ballerina/Module.md

```diff
@@ -0,0 +1,110 @@
+# Ballerina Zendesk connector
+
+Zendesk is a customer service software company that provides a cloud-based customer support platform. It is designed to
```

Comment:
We can link the official site to the `Zendesk` keyword here

Reply:
+1. Addressed in https://github.com/ballerina-platform/module-ballerinax-zendesk/pull/3/commits/b7f2b18f72493a877a10abd3b9b2c6b58aa971be